### PR TITLE
fix: Add toasts and error messages for wallet connection

### LINKED ISSUE #265: Swap interface does not have toasts and error messages for wallet connection
**Bug Description**
Swap interface does not have toasts and error messages for wallet connection

**Steps to Reproduce**

1. Click connect
2. Do some error like click metamask without having extension on your browser
3. No error message visible

**Expected Behavior**
It should have the same behaviour that rest of the app have

### PR DESCRIPTION
**Pull request by memoyil**
Fixes #265

### CODE REVIEW

**Comment by memoyil** [comment]
@hachiojidev You can re-review here

File: src/state/hooks.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import { useMemo } from 'react'
```
Comment (RabbitDoge) [inline src/state/hooks.ts:1]: why this file is not in `hooks` folder ?

File: src/state/hooks.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import { useDispatch } from 'react-redux'
+
+import {
+  push as pushToast,
+  remove as removeToast,
+  clear as clearToast
+} from './actions'
+
+// Toasts
+// eslint-disable-next-line import/prefer-default-export
```
Comment (RabbitDoge) [inline src/state/hooks.ts:13]: Why do you disable , instead of using `export default` ?

**Review by RabbitDoge — CHANGES_REQUESTED** [review]

**Review by lerog17 — APPROVED** [review]

**Review by lerog17 — APPROVED** [review]

File: src/state/hooks.ts (hunk 1 above)
Comment (memoyil) [reply src/state/hooks.ts]: moved

File: src/state/hooks.ts (hunk 2 above)
Comment (memoyil) [reply src/state/hooks.ts]: done

**Comment by lerog17** [comment]
Hi, Thank you for helping me please explain ut to me how...sorry, but i
don't know what's happening to me i keep on reading it but i can't excute
it...please keep in touch.

**Comment by Chef-Chungus** [comment]
> Hi, Thank you for helping me please explain ut to me how...sorry, but i don't know what's happening to me i keep on reading it but i can't excute it...please keep in touch.
> […](#)

Not sure what you're talking about, can you please create a github issue if it's not related to this PR

**Comment by memoyil** [comment]
@lerog17 You got some errors? I didnt get any

**Review by lerog17 — APPROVED** [review]

**Review by lerog17 — APPROVED** [review]

**Review by RabbitDoge — APPROVED** [review]

**Pull request merged**